# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/pointerevents/pointerup_after_pointerdown_target_removed.html.ini
# Commit: 0c3a2df68be4
# Full Hash: 0c3a2df68be4e59a46e89640b95e3f8ba7082a60
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2024-03-12 09:25:52
# Regressor Bug: 1695574
# File Overlap Count: 1
# Description:
#   Bug 1695574 - Make `PresShell::EventHandler::HandleEventUsingCoordinates` keep handling touch events after preceding pointer event target is removed r=smaug,edgar,dom-core
#   
#   As far as investigating the behavior of the other browsers, touch events are
#   dispatched the same target as the preceding `pointerdown` even after the target
#   is removed from the DOM tree.  However, the following pointer events are
# ==============================================================================

diff -r bdee0bed3f5c -r 0c3a2df68be4 testing/web-platform/meta/pointerevents/pointerup_after_pointerdown_target_removed.html.ini
--- a/testing/web-platform/meta/pointerevents/pointerup_after_pointerdown_target_removed.html.ini	Tue Mar 12 00:54:41 2024 +0000
+++ b/testing/web-platform/meta/pointerevents/pointerup_after_pointerdown_target_removed.html.ini	Tue Mar 12 01:03:44 2024 +0000
@@ -1,8 +1,3 @@
 [pointerup_after_pointerdown_target_removed.html?pen]
   [pointerup event from pen fired after pointerdown target is removed]
     expected: FAIL
-
-
-[pointerup_after_pointerdown_target_removed.html?touch]
-  [pointerup event from touch fired after pointerdown target is removed]
-    expected: FAIL